CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Move default branch from master to main
 
+### 2.2.0
```

Comment:
Let's just put this in with to 3.0.0 changes

Reply:
Should we merge #214 first? The PaymentOptions tests need to be rewritten with the new ctor

Reply:
Oh yes, That hasn't been merged yet. No, it's fine then. Let's keep to 2.2.0